### ISSUE TITLE
swtich over dev.rebuildingakidney.org to use new globus auth

### ISSUE THREAD

**Issue by hongsudt** [informatics-isi-edu/chaise#268]
This is needed before we can make changes to the rbk access control policy and the rbk frontend.

**Comment by karlcz** [comment]
It turns out I need to get some new client-side settings from Kyle for this.

**Comment by karlcz** [comment]
At this rate I'm not sure we're going to get any access prior to the new whitelist rolling out next week.

**Comment by karlcz** [comment]
This is done and you login via the new globus_auth method now.  I think this exposes a viewer app problem now, where its ACLs need to be checked against the new session attributes content.

**Comment by karlcz** [comment]
Reassigning to @hongsudt for review of RBK functions.  You might also want @ljpearlman to review the specifics of the webauthn config on the backend and in the provisioning recipe.

**Comment by hongsudt** [comment]
@ljpearlman can you please review the webauthn config and provisioning recipe?

**Comment by ljpearlman** [comment]
The webauthn config looks okay if the intention was to send people to /chaise/logout (instead of the main RBK page) post-logout. If not, the `default_logout_path` should be changed to whatever you want it to be.

**Comment by hongsudt** [comment]
@ljpearlman on rbk-staging, upon logout, the globus page is shown, not the chaise logout.

**Comment by svoinea** [comment]
This is already done on both rbk-dev and rbk-staging, as well as in their recipes.

**Comment by hongsudt** [comment]
@svoinea the link is still wrong. I don't want it to go back to the search app which requires a login page. It should go to project static link. For rbk, it is https://staging.rebuildingakidney.org/image-annotation/,

**Comment by hongsudt** [comment]
The link is now implemented correctly on rbk.
